### ISSUE TITLE
Slack Ending Support For Browser on September 1, 2021

### ISSUE THREAD

**Issue by jonmchan** [meetfranz/franz#2059]
I received a popup today that slack will be deprecating support for this browser. The user agent has to be updated. Unfortunately, dismissed the popup without taking a screenshot. Is the browser engine up to date? It could pose a security threat if Franz is using a browser from a few months or years ago.

**Comment by dtrucs** [comment]
The content of the popup refers to https://slack.com/intl/en-fr/help/articles/1500001836081-Slacks-deprecation-schedules#browser-deprecations.

**Comment by stefanvs** [comment]
<img width="519" alt="Screen Shot 2021-08-17 at 15 06 26" src="https://user-images.githubusercontent.com/6349005/129731099-70cad7b0-f930-4fc6-9038-ef9b10dc716f.png">

**Comment by BrianGilbert** [comment]
Pretty sure this can be closed givenm slack is still working just fine

**Comment by Brian-Webb** [comment]
As of this morning, Franz is not properly loading Slack for me; instead, it is trying to launch the Slack desktop application over and over.

I did find this in the package.json file: https://github.com/meetfranz/franz/blob/2fa838ed9264ff5f900592a4d8597ba6a1ba2212/package.json#L175

It appears that this needs to be updated to 83 or higher to work with Slack. I am not sure how much this would break/affect with anything else utilizing the browser APIs.

**Comment by gyumi314** [comment]
Ah, finally. I was going nuts why it doesn't open anymore. Same issue for me, it tries to open the desktop slack app. I tried uninstalling, but then nothing happens after loading (open in browser does nothing).

**Comment by stevelacey** [comment]
Yeah I am here to report that at least one of my Slacks no longer works, I can't get it to sign in via browser or Franz, just keeps jumping me to the Slack app, and the `use Slack in your browser` link does nothing but again, launch the Slack app 😕 

I am now getting stuck at this screen for 1 out of 3 of my Franz Slacks:

```
Launching {{ name }}
Click "Open Slack" to launch the desktop app.
Not working? You can also use Slack in your browser.
```

**Comment by siparker** [comment]
I was on v5.6.1 Beta 2. the updater was not seeing a new version so I downloaded it directly from the site manually and Slack is now working again.
I was getting repeated windows popups to choose a program to open a slack file.
